README.md

```diff
@@ -52,7 +52,35 @@ Prerequisites
    $ sudo pacman -Syu arm-none-eabi-gcc
    ```
 
-1. Optional: libtock-c also includes support for building for RISC-V targets.
+   **Fedora**:
+
+   **<ins>For `arm`</ins>:**
+   ```
+   $ sudo dnf install arm-none-eabi-newlib arm-none-eabi-gcc-cs
+   ```
+   **<ins>For `risc-v`</ins>:**
+
+   **dnf** does not contain the `riscv-gnu-toolchain`, an alternative is to compile from source. Start with some of the tools we need to compile the source.
+   ```
+   $ sudo dnf install make automake gcc gcc-c++ kernel-devel  texinfo expat expat-devel
+   $ sudo dnf group install "Development Tools" "C Development Tools and Libraries"
+   ```
+   Get `riscv-gnu-toolchain`,  [summarised instructions as stated here](https://github.com/riscv-collab/riscv-gnu-toolchain/blob/master/README.md)
+   ```
+   $ git clone https://github.com/riscv/riscv-gnu-toolchain
+   $ cd riscv-gnu-toolchain/
+   ```
+   **Note: add /opt/riscv/bin to your PATH**, then,
+   ```
+   $ ./configure --prefix=/opt/riscv --enable-multilib
+   ```
+   `--enable-multilib` ensures that "the multilib compiler will have the prefix riscv64-unknown-elf- or riscv64-unknown-linux-gnu- but will be able to target both 32-bit and 64-bit systems."
+   ```
+   $ sudo make linux         [might need elevated privileges writing to `/opt/`]
+   ```
+    After the the source has been compiled and copied to `/opt/riscv` and `/opt/riscv/bin`has appended to the PATH, the toolchain is ready to be used.
```

Comment:
Sorry! I was unclear. I meant to move all of this stuff into the next enumerated item below. You can take a look at the rendered view here: https://github.com/tock/libtock-c/blob/0614a609de1e9e19f4fa163aef5f1a553f656c7d/README.md which makes it more clear.

This item (3. in the rendered view) is instructions for all OSes for ARM.

The one below that starts "Optional: libtock-c also includes support for building for RISC-V targets." (4. in the rendered view) has instructions for all the OSes for RISC-V. So the Fedora instructions for RISC-V should be split from the ARM instructions and moved down there.

Reply:
Gotcha, fixed. Thanks for that!